js-user-data-manipulation/index.js

```diff
@@ -0,0 +1,150 @@
+// user name showing
+
+const userHeader = document.querySelector('h1');
+const input = document.querySelector('.user-form__input');
+const btn = document.querySelector('.user-form__btn');
+let username;
+
+try {
+  username = localStorage.getItem('username');
+  setUserHeader();
+} catch (error) {
+  console.log(error);
+}
+
+btn.addEventListener('click', event => {
+  event.preventDefault();
+  console.log(input.value);
+  username = input.value;
```

Comment:
Hmm. This is not a good approach to use some inputs from outside. Will be better use value from event like this `event.currentTarget.value`. to avoid some side effects